### PR TITLE
OSDOCS-2681: RN for MetalLB 4.10

### PR DESCRIPTION
**Pull request by mikemckiernan**
Please review the [Enhancements to MetalLB load balancing](https://deploy-preview-40351--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-10-release-notes#ocp-4-10-networking).

I'm happy to add faves.  I recorded what I felt were the highlights.

### CODE REVIEW

**Comment by mikemckiernan** [comment]
/assign @msherif1234 

PTAL and let me know if there are additional enhancements to highlight.

**Comment by mikemckiernan** [comment]
/assign @asood-rh 

PTAL and check that I haven't highlighted any features that you have not reviewed.  Thank you.

**Comment by jboxman-rh** [comment]
cc @jboxman-rh

**Comment by skrthomas** [comment]
@asood-rh Could you please review this RN?

File: release_notes/ocp-4-10-release-notes.adoc [hunk 1]
```diff
@@ -134,6 +134,20 @@ A new `--node-selector` argument provides a way to identify which nodes you are
...
 
+[id="ocp-4-10-networking-metallb"]
+==== Enhancements to MetalLB load balancing
+
+The following enhancements to MetalLB and the MetalLB Operator are included in this release:
+
+* Support for Border Gateway Protocol (BGP) is added.
+* Support for Bidirectional Forwarding Detection (BFD) in combination with BGP is added.
+* Support for IPv6 and dual-stack networking is added.
+* Support for specifying a node selector on the `speaker` pods is added so you can control which nodes are used for load balancer IP addresses. This enhancement applies to layer 2 mode and BGP mode.
```
Comment (asood-rh) [inline release_notes/ocp-4-10-release-notes.adoc:145]: Could you add rephrase the sentence to this?
Support for specifying a node selector on the `speaker` pods is added so you can control which nodes are used for **advertising** load balancer **service** IP addresses. This enhancement applies to layer 2 mode and BGP mode

Comment (skrthomas) [reply release_notes/ocp-4-10-release-notes.adoc:145]: Yep, I can add those words :) New [PR](https://github.com/openshift/openshift-docs/pull/41476) for this RN where I incorporate your changes.

**Pull request closed without merging**